packages/grafana-data/src/events/types.ts

```diff
@@ -62,11 +63,6 @@ export interface EventBus {
    */
   publish<T extends BusEvent>(event: T): void;
 
-  /**
-   * Subscribe to single event
-   */
-  subscribe<T extends BusEvent>(eventType: BusEventType<T>, handler: BusEventHandler<T>): Unsubscribable;
```

Comment:
@torkelo, do we really need/want this variation on the EventBus interface?  handler is pretty close to `PartialObserver<T>` so :shrug: 

I moved it to the "legacy" section so existing code will keep working, but this will discourage use in new panels

Reply:
what would they use instead? This is the function I think all new users of the EventBus is using

Reply:
I know this function is not strictly needed and syntactic sugar for the subscribing to an observable, but makes the consumer code a lot cleaner, 

```ts
 this.subs.add(panel.events.subscribe(RefreshEvent, this.onRefresh));
 this.subs.add(panel.events.subscribe(RenderEvent, this.onRender));
``` 

has to be written as

```ts
this.subs.add(panel.events.getStream(RefreshEvent).subscribe({
     next: this.onRefresh
}));
this.subs.add(panel.events.getStream(RenderEvent).subscribe({
     next: this.onRender
}));
```

the subscribe function is really there just to simplify consumption without having to look up how Observables work. But if we want one way only, and a simpler interface then it can be removed.